android/java/org/chromium/chrome/browser/crypto_wallet/util/WalletUtils.java

```diff
@@ -86,6 +86,11 @@ public static boolean accountIdsEqual(AccountId left, AccountId right) {
     }
 
     public static boolean accountIdsEqual(AccountInfo left, AccountInfo right) {
```

Comment:
Just a suggestion but if we want to receive help from the IDE/lint checks, we can mark both parameters as `nullable`.

```java
    public static boolean accountIdsEqual(@Nullable final AccountInfo left, @Nullable final AccountInfo right) {
```